tests/run/worktree/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+        rlRun "cp -rf data/* $tmp"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Simple"
+        rlRun "pushd $tmp/simple"
+        rlRun "tmt run"
+        rlRun "tmt run --last report -vvv"
```

Comment:
What about `tmt run -ar report -vvv` in one line and use `-r` to remove the workdir?